.github/workflows/release_windows.yml

```diff
@@ -88,24 +88,17 @@ jobs:
           $latestYml.files[0].size = $size
           $latestYml | ConvertTo-Yaml | Set-Content -Path signed/latest.yml
 
-      - name: Create draft release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: Release ${{ github.ref }}
-          draft: true
-          prerelease: false
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
 
-      - name: Upload release asset
+      - name: Upload asset
         id: upload-release-asset
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
         with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
           asset_path: signed/HyperPlay-Setup.exe
           asset_name: HyperPlay-Setup.exe
```

Comment:
can we change these names on this branch? or is it out of scope?

Reply:
I remember now why I needed to rename it like this.,
I was unable to make powershell to understand something like `HyperPlay*.exe` when uploading the file.
Tried in several ways and did not work.

Reply:
Here on this part:
https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/blob/d0d1849d5698928c5022c87750bb3e49521dde49/.github/workflows/release_windows.yml#L82

Reply:
can you use something like `HyperPlay-${{ github.ref }}-Setup-x64.exe`?

Reply:
or set the `${{ github.ref }}` as an env variable for that step and use that env variable in the exe string name

Reply:
I fixed the naming already by getting the result of the listing. Because then we won't risk breaking the flow in case we update the artifact name in the future.